webpack.config.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+};
```

Comment:
### [OPTIONAL]

Great work adding webpack. But you have left out some rules. Kindly configure your webpack well. Add rules for the styles.